webapp/src/pages/Users/Users.css

```
@@ -1,21 +1,92 @@
 .users {
   margin: 0;
-  padding: 0;
+  padding: var(--padding-top-bottom) var(--padding-sides);
   min-height: 100vh;
   background-color: var(--color-background);
   color: var(--color-light);
+  grid-template-rows: auto auto 1fr auto;
```

```suggestion
  height: 100vh;
  background-color: var(--color-background);
  color: var(--color-light);
  display: grid;
  grid-template-columns: 1fr;
  grid-template-rows: auto auto 1fr auto;
  gap: var(--row-gap) var(--column-gap);
```

suggestion: We could set the `height` to `100vh` and use a grid display here. This way, the `users-rows` will occupy the remaining space automatically, and we could remove the `margin` and `height` of the items (`users-search`, `users-rows` and `users-navigation`)

---

what do you think about setting the number of columns in mobile to 4? Not that we need them here, but it works equally, and like that we can use this style sheet as boilerplate for most of the rest of the pages

---

Sure, as you see fit.

---

No, it's paying for something we are not using, and the change will be trivial, when we do need it. I prefer your choice.